src/Hw.css

```
@@ -0,0 +1,39 @@
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 50px;
+}
+
+.arrayBox {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  margin-right: 15px;
+
+  width: 45px;
+  height: 55px;
+
+  font-size: 18px;
+  border: 2px solid black;
+}
+
+.increase {
+  all: unset;
+  text-align: center;
+  font-weight: bolder;
+  background-color: #7ed6df;
+  width: 60px;
+  height: 60px;
+  margin-right: 15px;
+}
+
+.decrease {
+  all: unset;
+  text-align: center;
+  font-weight: bolder;
+  background-color: #badc58;
+  width: 60px;
+  height: 60px;
+}
```

```suggestion
.someButton {
  all: unset;
  text-align: center;
  font-weight: bolder;
  width: 60px;
  height: 60px;
}

.increase {
  background-color: #7ed6df;
  margin-right: 15px;
}

.decrease {
  background-color: #badc58;
}
```

공통되는 부분은 한 클래스에 묶어서 사용할 수 있다는 거 알고 계시면 좋을 거 같아요 ~!

```jsx
<button className={'someButton decrease'} />
```

---

대박이군요 .. 🤩